tests/test_util.py

```diff
@@ -48,34 +49,49 @@ class BufferIteratorToLineIteratorTestCase(TestCase):
 
     def test_empty(self):
         self.assertEqual(
-            list(buffer_iterator_to_line_iterator(_ for _ in ())),
+            list(yield_lines(_ for _ in ())),
             [])
 
     def test_buffered_lines(self):
         self.assertEqual(
-            list(buffer_iterator_to_line_iterator(chunk for chunk in
-                                                  [b'The quick\nbrown fox\nju',
-                                                   b'mped over\nthe lazy\ndog',
-                                                   b's.\n'])),
+            list(yield_lines(chunk for chunk in
```

Comment:
Cool. mrjob uses PEP8 for style (see the [release checklist](https://github.com/Yelp/mrjob/wiki/Release-Checklist).

Does Yelp have a maximum line length?